client/src/components/DetailsEvent/index.js

```diff
@@ -1,13 +1,180 @@
+                  <label className="answer">{org_name}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Hall cost:</label>
+                  <label className="answer">{price}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Number of persons:</label>
+                  <label className="answer">{capacity}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">Start Date:</label>
+                  <label className="answer">{start_ddmmyyyy}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">End Date:</label>
+                  <label className="answer">{end_ddmmyyyy}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">note:</label>
+                  <label className="answer">{note}</label>
+                </div>
+                <h2 className="h2">Lunch</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{lunch_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">time:</label>
+                  <label className="answer">{lunch_time}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price per person:</label>
+                  <label className="answer">{lunch_price}</label>
+                </div>
+                <h2 className="h2">Coffee</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{coffee_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">time:</label>
+                  <label className="answer">{coffee_time}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price per person:</label>
+                  <label className="answer">{coffee_price}</label>
+                </div>
+                <h2 className="h2">Equipment</h2>
+                <div className="labels_container">
+                  <label className="title">Description:</label>
+                  <label className="answer">{equipment_note}</label>
+                </div>
+                <div className="labels_container">
+                  <label className="title">price:</label>
+                  <label className="answer">{equipment_price}</label>
+                </div>
+                {this.state.status && <input type="submit" value="OK" />}
```

Comment:
Could pull all of this out into a function above the `render` function, which takes status, something like (syntax might be wrong):

```js
renderButtons = status => { 
  const button = status ?
   <input type="submit" value="OK" />
   : <input type="submit" value="Approve" />

return (
  { button}
 <input type="button" value="Cancel" onClick={this.cancel} />
)
}
```

Then in the render you just call this function and pass status, ```this.renderButtons(this.state.status)```

Reply:
Also I'm not sure about what happens here, do you only show the cancel button if the status doesn't exist? Maybe this is just me being confused, what does `status === true` actually mean? That it has been Approved?

Reply:
yeah if status === 1 "true" that means this event has been approved so I will not show this button anymore, and will show the button with "ok" as it form to show details only.